### PR TITLE
Mix silence in with sdk audio

### PR DESCRIPTION
**Pull request by frostbyte73**

### CODE REVIEW

File: pkg/pipeline/input/builder/audio.go [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+		return err
+	}
+	a.testSrc = []*gst.Element{audioTestSrc, audioCaps}
+
+	audioMixer, err := gst.NewElement("audiomixer")
+	if err != nil {
+		return err
+	}
+	// set latency slightly higher than max audio appsrc latency
+	if err = audioMixer.SetProperty("latency", uint64(41e8)); err != nil {
```
Comment (davidzhao) [inline pkg/pipeline/input/builder/audio.go:233]: should these latency values be constants or configurable?

File: pkg/pipeline/input/builder/audio.go [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+		return errors.ErrNotSupported(string(p.AudioCodec))
+	}
+
+	return nil
+}
+
+func getCapsFilter(p *params.Params) (*gst.Element, error) {
+	var caps *gst.Caps
+	switch p.AudioCodec {
+	case params.MimeTypeOpus:
```
Comment (davidzhao) [inline pkg/pipeline/input/builder/audio.go:278]: should we also set websocket / raw output to two channel and 48k for consistency for real-time workflows?

**Review by davidzhao — APPROVED** [review]
👍

File: pkg/pipeline/input/builder/audio.go (hunk 2 above)
Comment (frostbyte73) [reply pkg/pipeline/input/builder/audio.go:278]: it's already doing that, audio codec is opus in that case with output type raw

**Pull request merged**